packages/light-apps/tsconfig.json

```diff
@@ -19,6 +19,6 @@
   },
   "extends": "../../tsconfig",
   "include": [
-    "src"
+    "src", "../accounts-app/src/ManageAddresses"
```

Comment:
Is this needed?

Reply:
ah that must have been an auto correct import paths thing